packages/xdl/package.json

```diff
@@ -72,6 +72,8 @@
     "mv": "2.1.1",
     "ncp": "2.0.0",
     "node-forge": "0.7.6",
+    "p-map": "^3.0.0",
+    "p-retry": "^4.1.0",
```

Comment:
Please keep the dependencies' versions strict.

Reply:
Good call